config/fund_loader_config/digital_planning/dpi_r2.py

```diff
@@ -0,0 +1,149 @@
+    },
+    {
+        "section_name": {"en": "2. Your skills and experience", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.3",
+    },
+    {
+        "section_name": {"en": "2.1 Your skills and experience", "cy": ""},
+        "form_name_json": {"en": "your-skills-and-experience-dpi", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.3.1",
+    },
+    {
+        "section_name": {"en": "2.2 Roles and recruitment", "cy": ""},
+        "form_name_json": {"en": "roles-and-recruitment-dpi", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.3.2",
+    },
+    {
+        "section_name": {"en": "3. About your project", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4",
+    },
+    {
+        "section_name": {"en": "3.1 Engaging the ODP community", "cy": ""},
+        "form_name_json": {"en": "engaging-the-odp-community-dpi", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.1",
+    },
+    {
+        "section_name": {"en": "3.2 Engaging the organisation", "cy": ""},
+        "form_name_json": {"en": "engaging-the-organisation-dpi", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.2",
+    },
+    {
+        "section_name": {"en": "3.3 Dataset information", "cy": ""},
+        "form_name_json": {"en": "", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.3",
+    },
+    {
+        "section_name": {"en": "4. Future work", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.5",
+    },
+    {
+        "section_name": {"en": "4.1 Future work", "cy": ""},
+        "form_name_json": {"en": "future-work-dpi", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.5.1",
+    },
+    {
+        "section_name": {"en": "5. Declarations", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.6",
+    },
+    {
+        "section_name": {"en": "5.1 Declarations", "cy": ""},
+        "form_name_json": {"en": "declarations-dpi", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.6.1",
+    },
+]
+
+fund_config = {
+    "id": DPI_FUND_ID,
+    "name_json": {
+        "en": "Digital Planning Improvement Fund",
+        "cy": "",
+    },
+    "title_json": {
+        "en": "funding to adopt modern planning practices",
+        "cy": "",
+    },
+    "short_name": "DPI",
+    "description_json": {"en": "", "cy": ""},
+    "welsh_available": False,
+    "owner_organisation_name": "Department for Levelling Up, Housing and Communities",
+    "owner_organisation_shortname": "DLUHC",
+    "owner_organisation_logo_uri": DLUHC_LOGO_PNG,
+}
+
+round_config = [
+    {
+        "id": DPI_ROUND_2_ID,
+        "fund_id": DPI_FUND_ID,
+        "title_json": {"en": "Round 2", "cy": ""},
+        "short_name": "R2",
+        "opens": DPI_R2_OPENS_DATE,
+        "deadline": DPI_R2_DEADLINE_DATE,
+        "assessment_deadline": DPI_R2_ASSESSMENT_DEADLINE_DATE,
+        "prospectus": DPI_PROSPECTS_LINK,
+        "privacy_notice": "",  # to be added
```

Comment:
Could make a proper TODO so linters pick it up, and give it an assigned user maybe?

```suggestion
        "privacy_notice": "",  # TODO(tferns): to be added
```